pkg/controllers/status/cluster_status_controller.go

```diff
@@ -257,19 +268,21 @@ func (c *ClusterStatusController) buildInformerForCluster(cluster *clusterv1alph
 	for _, gvr := range gvrs {
 		if !singleClusterInformerManager.IsInformerSynced(gvr) {
 			allSynced = false
-			singleClusterInformerManager.Lister(gvr)
+			if _, err := singleClusterInformerManager.Lister(gvr); err != nil {
+				klog.Errorf("Failed to get the lister for gvr %s: %v", gvr.String(), err)
+			}
 		}
 	}
 	if allSynced {
 		return singleClusterInformerManager, nil
 	}
 
-	c.InformerManager.Start(cluster.Name)
+	c.TypedInformerManager.Start(clusterClient.ClusterName)
```

Comment:
Why genericInformerManager not start?

Reply:
there is no resource informer added in genericInformerManager at the time, so if genericInformerManager start, no informer will start. I think it should start when a new informer is created.

Reply:
I think it's strange that we do not start the informermanager but stop it in the same controller.

Reply:
got it.